akvo/rsr/models/result/indicator_period_disaggregation.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+from django.db import models
+from akvo.rsr.mixins import TimestampsMixin
+from django.utils.translation import ugettext_lazy as _
+
+
+class IndicatorPeriodDisaggregation(TimestampsMixin, models.Model):
+    """
+    Model to hold aggregated disaggregation updates
+    """
+
+    period = models.ForeignKey(
```

Comment:
Would it be worth making an abstract model where we have all these fields? And use that everywhere we need these fields?

Reply:
Yeah, I think that's good enough. May be also use that for the `IndicatorPeriodData` ? This would make the updates, and the disaggregations also be somewhat consistent.

Reply:
Yes, it can also used in `IndicatorPeriodData`, the mixin's name is still appropriate to use there.

Reply:
Can we just make it `IndicatorValueMixin` or `IndicatorUpdateMixin` ?